inspector/src/components/popupComponent.tsx

```diff
@@ -36,6 +36,12 @@ export class PopupComponent extends React.Component<IPopupComponentProps, { isCo
         this.setState({ isComponentMounted: true });
     }
 
+    onBeforeLoadListener = () => {

```

Comment:
should probably be renamed to unBeforeUnloadListener :-)

Reply:
UNLoad... I missed by two letters! :-)